src/pages/workspaces/workspace/workflows/LaunchAnalysisModal.js

```diff
@@ -114,13 +125,21 @@ const LaunchAnalysisModal = ({
           p(['Note that metadata about this run will be stored in the US.'])
         ])]) : 'Loading...'
     ]),
-    div({ style: { margin: '1rem 0' } }, [
-      'This will launch ', b([entityCount]), ` analys${entityCount === 1 ? 'is' : 'es'}`,
-      '.',
-      type === processMergedSet && entityCount !== mergeSets(selectedEntities).length && div({
-        style: { fontStyle: 'italic', marginTop: '0.5rem' }
-      }, ['(Duplicate entities are only processed once.)'])
+    warnDuplicateAnalyses ? div({ style: { margin: '1rem 0', ...fullWidthWarningStyle } }, [
+      icon('warning-standard', { size: 16, style: { color: colors.warning(), marginRight: '0.5rem' } }),
+      'Warning! This will launch ',
+      b([entityCount]),
+      span({ style: { fontWeight: 'bold', textDecoration: 'underline' } }, [' duplicate']),
+      entityCount === 1 ? ' analysis' : ' analyses',
+      warnDuplicateAnalyses && ' of the same set of constant inputs.'
```

Comment:
Now that we have two code paths, these conditionals aren't needed, so this could be:
```suggestion
      ' analyses of the same set of constant inputs.'
```